components/automate-chef-io/content/docs/restore.md

```diff
@@ -23,15 +22,14 @@ Restore Chef Automate from a [fileystem backup]({{< ref "restore.md#restore-from
     ```
 
 1. **Filesystem backups** require access for Chef Automate to a backup directory in the [configured location]({{< ref
-"backup.md#backup-to-a-filesystem" >}}):
+"backup.md#backup-to-a-filesystem" >}}).
+Ensure access for the backup type used:
 
      1. To restore a network-attached filesystem backup, mount the shared backup directory to the same mount point configured at the time of the backup.
-     1. To restore a backup directory that is not a network-attached filesystem, copy the backup directory to the location that was [configured]({{< ref
-"backup.md#backup-to-a-filesystem" >}}) at the time of the backup.
+     1. To restore a backup directory that is not a network-attached filesystem, copy the backup directory to the [configured location]({{< ref "backup.md#backup-to-a-filesystem" >}}) at the time of the backup.
      1. For restoring a single-file backup archive, copy your archive to the restore host and extract it to the [configured backup directory]({{< ref "backup.md#backup-to-a-filesystem" >}}).
 
-1. **(different FQDN only)** To restore to a host whose FQDN differs from that of the host on which the backup was
-   taken, create a `patch.toml` file to provide at restore time that specifies the new FQDN:
+1. **(OPTIONAL)** To restore to a host with a different fully qualified domain name (FQDN) than that of the backup host, create a `patch.toml` file that specifies the new FQDN and provides it at restore time:
```

Comment:
If the user is restoring to a host whose fqdn is different from that of the original Automate install, then they do need to change the configured fqdn to get a working system.

Reply:
@yzl That's a good detail to know. I'm not quite sure where you're going with it. Are you suggesting for this to be inserted in line 32 or are you suggesting to amend a detail in line 32?

Reply:
@mjingle The directions in line 32 are not optional for someone who is restoring to a host with a different FQDN, so I'd prefer that the bolded text not encourage someone in that situation to skip reading.

Reply:
@yzl Okay! Now I'm following. Thank you!  I had interpreted "**(different FQDN only)**" as an optional step because my fictional reader could potentially be confused if we're listing required pre-reqs in this section but there is a conditional pre-req also included.

So, this part needs to be rephrased to say something along the lines of "If restoring to a host with a different..." and the bolded wording needs to be removed. (I can do that with my promised link fix.) That way, folxs will still read it thinking it's a required pre-req but will be able to determine that it is conditional and it may not apply to their situation.